services/shotgrid_common/validate.py

```diff
@@ -0,0 +1,313 @@
+""" Validation
+"""
+import collections
+
+import ayon_api
+from ayon_api.entity_hub import EntityHub
+
+
+import constants
+from utils import get_logger, get_sg_project_enabled_entities
+
+
+def check_project_disabled_entities(
+        ay_project,
```

Comment:
Formatting seems weird (double tab for arguments, tab for `)`). Would this make Ruff mad?